app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,86 @@
+class TweetsController < ApplicationController
+  get '/tweets' do
+    if !session[:id].nil?
+      @user = User.find_by(session[:id])
```

Comment:
we should use our new `current_user` for this line of code, and for any other `session[:id]` checks in your application.